src/bundles/config.js

```diff
@@ -74,4 +86,16 @@ function getURLFromAddress (name, config) {
   }
 }
 
+const checkIfGatewayUrlIsAcessible = memoizee(async (url) => {
+  try {
+    const { status } = await fetch(
+    `${url}/ipfs/bafybeiczsscdsbs7ffqz55asqdf3smv6klcw3gofszvwlyarci47bgf354`
+    )
+    return status === 200
+  } catch (e) {
+    console.error(e)
```

Comment:
Additional context will be super useful here:
```suggestion
    console.error(`Unable to use Gateway at ${url}, the public one will be used as a fallback`, e)
```

Reply:
Could I butt in to suggest small language change?
```suggestion
    console.error(`Unable to use the gateway at ${url}. The public gateway will be used as a fallback`, e)
```

Reply:
Thanks 🙏